modules/fedimint-wallet-server/src/lib.rs

```diff
@@ -1000,6 +1021,13 @@ pub struct Wallet {
     block_count_rx: watch::Receiver<Option<u64>>,
     /// Fee rate updated periodically by a background task
     fee_rate_rx: watch::Receiver<Feerate>,
+
+    /// Update to notify consensus item proposal about a change worth proposing
+    propose_citem_tx: watch::Sender<()>,
+    /// Consensus proposals will wait for this to update, wrapped in `Arc` and
+    /// `RwLock` for internal mutability.
+    propose_citem_rx: Arc<tokio::sync::RwLock<watch::Receiver<()>>>,
```

Comment:
Requiring an `Arc<RwLock<_>>` seems wrong. Wouldn't a good old [`Notify`](https://docs.rs/tokio/latest/tokio/sync/struct.Notify.html) do the trick here?

Reply:
`propose_consensus` has immutable `&self`. That's also the only using it. I guess I only need `RwLock<watch::Receiver<()>>`. I do not understand `Notify`, the documentation is confusing, and I'm not sure if it does what I want. I do know that `watch` works exactly as I want. :D

Reply:
Hmmm... OK... I guess it is better...